infra/config-store/src/main/java/com/simprints/infra/config/store/models/ProjectConfiguration.kt

```diff
@@ -41,3 +43,11 @@ fun ProjectConfiguration.isEventDownSyncAllowed(): Boolean =
 
 fun ProjectConfiguration.imagesUploadRequiresUnmeteredConnection(): Boolean =
     synchronization.up.simprints.imagesRequireUnmeteredConnection
+
+fun ProjectConfiguration.allowedAgeRanges(): List<AgeGroup> {
+    return listOf(
+        //Todo add face roc sdk ,
+        fingerprint?.secugenSimMatcher?.allowedAgeRange,
+        fingerprint?.nec?.allowedAgeRange
+    ).mapNotNull { it }
+}
```

Comment:
If you look at the [API definition ](https://app.swaggerhub.com/apis/CLOUDPLATFORMDEVS_1/bfsid-android-api/2024.2.0#/ProjectConfiguration) the `allowedAgeRange` field is always present, but it's empty when both start and end are equal to null